flock-addon/Makefile

```
@@ -0,0 +1,902 @@
+		S3_COMPAT_ACCESS_KEY="$$ACCESS_KEY" \
+		S3_COMPAT_SECRET_KEY="$$SECRET_KEY" \
+		S3_COMPAT_BUCKET="$$BUCKET"; \
+	echo "[flock-addon] uploading $(MODEL_ARCHIVE) to bucket $$BUCKET with key $$MODEL_HASH"; \
+	$(DOCKER) run --rm --network host --entrypoint /bin/sh \
+		-e MODEL_BASENAME="$$MODEL_BASENAME" \
+		-e MODEL_HASH="$$MODEL_HASH" \
+		-e BUCKET="$$BUCKET" \
+		-e ACCESS_KEY="$$ACCESS_KEY" \
+		-e SECRET_KEY="$$SECRET_KEY" \
+		-v "$$MODEL_DIR:/upload:ro" '$(MINIO_MC_IMAGE)' -ec '\
+		mc alias set local http://127.0.0.1:$(S3_COMPAT_PORT) "$$ACCESS_KEY" "$$SECRET_KEY" >/dev/null && \
```

checkmake (0.3.2)

[warning] Target body for "mc alias set local http" exceeds allowed length of 5 lines (58).

(maxbodylength)